deployability/modules/allocation/allocation.py

```diff
@@ -258,3 +259,33 @@ def __check_connection(inventory: models.InventoryOutput, attempts=30, sleep=30)
 
         logger.error(f'Connection attempts failed after {attempts} tries. Connection timeout.')
         return False
+
+    @staticmethod
+    def __get_auxiliar_files_path(path: Path, instance: Instance, file: str) -> Path:
+        """
+        Get the path of the auxiliar files.
+
+        Args:
+            path (Path): The path of the instance.
+            instance (Instance): The instance object.
+            file (str): The type of file.
+
+        Returns:
+            Path: The path of the auxiliar files.
+        """
+        if path is None:
+            return Path(os.path.join(instance.path, file + '.yml'))
+        if path.exists() and path.is_dir():
+            return Path(os.path.join(path, file + '.yml'))
+        if str(path).endswith('.yml') or str(path).endswith('.yaml'):
+            if file in str(path):
+                return path
+            elif file == 'inventory' and 'track' in str(path):
+                return Path(str(path).replace('track', 'inventory'))
+            elif file == 'track' and 'inventory' in str(path):
+                return Path(str(path).replace('inventory', 'track'))
```

Comment:
I don't think we should assume the error and auto-fix it if we validate and force that those files should have a specific name, then we can work with this, but we need to be sure not to accept false positives

We can also add an ID like `type: 'inventory'` in the inventory and track files, this will avoid checking names and content